src/porepy/utils/error.py

```diff
@@ -42,6 +45,10 @@ def grid_error(
         variable = [variable]
     if not isinstance(variable_dof, list):
         variable_dof = [variable_dof]
+    if normalizations is None:
+        normalizations = [lambda sol: np.linalg.norm(sol_ref, axis=0)] * len(variable)
+    elif not isinstance(normalizations, list):
+        normalizations = [normalizations]
```

Comment:
Is this intended as a helper for the case where you only consider one variable?
So, if you have multiple variables and wish to apply the same normalization for all of them, you are yourself responsible to do `[my_normalization] * len(variable)`?

Reply:
Indeed